client/client.go

```diff
@@ -717,13 +785,98 @@ func newWebsocketDialerTLS(conf Config) (*websocket.Dialer, error) {
 
 	dialer := websocket.Dialer{
 		NetDialTLSContext: func(_ context.Context, network string, addr string) (net.Conn, error) {
+			proxyUrl, err := getProxyUrlFromHostPort(addr)
+			if err != nil {
+				log.Errorf("Failed to get http-proxy configurations during websocket dial: %v", err)
```

Comment:
I would argue that if this fails we want the whole operation to fail (not to continue dialing with no proxy). This is what I think a user expects, so that he/she can see what is going on and possibly amend the env variables.